<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>placeProjectWizard</class>
 <widget class="QWizard" name="placeProjectWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Place Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/creepy/user</normaloff>:/creepy/user</iconset>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="placeProjectWizardPage1">
   <property name="title">
    <string>Step 1 - Set Project Metadata</string>
   </property>
   <property name="subTitle">
    <string>Add project related information</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="placeProjectDescriptionValue">
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="placeProjectNameValue">
       <property name="placeholderText">
        <string>Add a name for your project</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="placeProjectNameLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Project Name </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="placeProjectKeywordsValue">
       <property name="placeholderText">
        <string>Add comma seperated keywords for your project</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="placeProjectDescriptionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="personProkectKeywordsLabel">
       <property name="text">
        <string>Keywords</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="placeProjectWizardPage2">
   <property name="title">
    <string>Step 2 - Set the target</string>
   </property>
   <property name="subTitle">
    <string>Select the Point Of Interest and the distance around which you want to get results</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="containerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Drop a &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ff0000;&quot;&gt;pin&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; on the map for your point of interest&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebView" name="webView" native="true">
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="searchAddressLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="searchAddressLabel">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchAddressInput">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchAddressButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="controlsContainerLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="radiusLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Distance from the POI :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="radiusSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="radiusUnitComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="placeProjectWizardPage3">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>841</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="containerLayout_2">
     <item>
      <widget class="QLabel" name="placeProjectSearchInLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Search In&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="placeProjectAvailablePluginsListView"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="placeProjectWizardPage4">
   <property name="title">
    <string>Step 3 - Set Parameters</string>
   </property>
   <property name="subTitle">
    <string>Provide the necessary search parameters for the plugins you are using</string>
   </property>
   <widget class="QListView" name="placeProjectWizardSearchConfigPluginsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="searchConfiguration">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="placeProjectWizardPage5">
   <property name="title">
    <string>Step 4  - Finalize Project</string>
   </property>
   <property name="subTitle">
    <string>Click Finish to save the Project Configuration </string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../creepy/include/creepy_resources.qrc"/>
 </resources>
 <connections/>
</ui>
